timed/projects/models.py

```diff
@@ -242,3 +247,18 @@ class TaskAssignee(models.Model):
     is_resource = models.BooleanField(default=False)
     is_reviewer = models.BooleanField(default=False)
     is_manager = models.BooleanField(default=False)
+
+
+@receiver(pre_save, sender=Project)
+def update_billed_flag_on_reports(sender, instance, **kwargs):
+    """Update billed flag on all reports from the updated project.
+
+    Only update reports if the billed flag on the project was changed.
```

Comment:
I would like to see a docstring that explains the logic/intention behind the code, so that readers in the future understand why we're doing this. Just copy a fragment from the PR here

Reply:
Added a short explanation. I hope it helps understanding the logic behind it. Let me know if it's enough or not.